e2e/tests/nightly/genomic-extraction-to-vcf.spec.ts

```diff
@@ -22,17 +22,17 @@ import { takeScreenshot } from 'utils/save-file-utils';
 import expect from 'expect';
 import { range } from 'lodash';
 
-// 70 minutes. Test could take a long time.
+// 60 minutes. Test could take a long time.
 // Since refresh token expires in 60 min. test may fail if running takes longer than 60 min.
-jest.setTimeout(70 * 60 * 1000);
+jest.setTimeout(60 * 60 * 1000);
 
 describe('Genomics Extraction Test', () => {
   beforeEach(async () => {
     await signInWithAccessToken(page);
   });
 
   const waitForCompletionMaxTime = 60 * 60 * 1000;
```

Comment:
This limit should be shorter than the overall timeout. By the time we reach this point in the code, we don't have 60 minutes left